code/modules/holodeck/computer.dm

```diff
@@ -25,44 +31,76 @@
 	idle_power_usage = 10
 	active_power_usage = 50
 
+	//new vars
+	///what access type this holodeck has, used to specify programs for another holodeck that others cant load.
+	var/holodeck_access = STATION_HOLODECK
+
+	///what area type this holodeck loads into. linked turns into the nearest instance of this area
+	var/area/mappedstartarea = /area/holodeck/rec_center
```

Comment:
could you use `snake_case` instead of `nospacescase` ?

Reply:
mapped_start_area now